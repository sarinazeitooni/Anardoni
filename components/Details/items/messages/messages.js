const messages ={
    free : 'رایگان',
    information : 'اطلاعات',
    linkTitle : 'لینک های مفید',
    whatsNew : 'جدید چه خبره؟',
    swipperTitle : 'پیش نمایش',
    version : ' نسخه ',
    versionHistory : 'نمایش تاریخچه ورژن ها',
    more: 'بیشتر',
    close : 'بستن',
    description: 'توضیحات',
    backBtn : 'برگشت',
    getbtn : 'دریافت',
    age: 'سن',
    feedback : 'بازخورد'
};
export default messages;