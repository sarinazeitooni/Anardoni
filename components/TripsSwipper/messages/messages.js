const messages = {
    trips: 'سفرهای ضروری',
    more : 'بیشتر',
    getButton : "دریافت",
};
export default messages;