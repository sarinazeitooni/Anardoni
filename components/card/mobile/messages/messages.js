const messages = {
    getButton : 'دریافت',
};
export default messages;