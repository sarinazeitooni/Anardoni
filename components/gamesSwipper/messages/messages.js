const messages = {
    games : ' بازی ها',

};
export default messages;