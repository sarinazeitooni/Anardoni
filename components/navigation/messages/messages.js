const messages = {
    updates : 'آپدیت ها',
    home : ' خانه ',
    apps : ' برنامه ها ',
    games : ' بازی ها ',
    categories : 'دسته بندی ها',
    search : 'جستجو',
    signup : ' ثبت نام ',
    login: ' ورود '
};
export default messages;