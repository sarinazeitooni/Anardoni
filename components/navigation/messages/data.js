import messages from './messages';
import HomeTwoToneIcon from '@material-ui/icons/HomeTwoTone';
import SearchTwoToneIcon from '@material-ui/icons/SearchTwoTone';
import LayersIcon from '@material-ui/icons/Layers';
import WhatshotIcon from '@material-ui/icons/Whatshot';
import CategoryTwoToneIcon from '@material-ui/icons/CategoryTwoTone';
const Data = [
    {
        number: 'first',
        text: messages.home,
        icon : <HomeTwoToneIcon/>,
        href : '/'
    },
    {
        number: 'second',
        text: messages.apps,
        icon : <LayersIcon/>,
        href : '#apps'
    },
    {
        number: 'third',
        text: messages.games,
        icon : <WhatshotIcon/>,
        href : '#games'
    },
    {
        number: 'forth',
        text: messages.categories,
        icon : <CategoryTwoToneIcon/>,
        href : '#categories'
    },
    {
        number: 'fifth',
        text: messages.search,
        icon : <SearchTwoToneIcon/>,
        href : '#search'
    }
]
export default Data;