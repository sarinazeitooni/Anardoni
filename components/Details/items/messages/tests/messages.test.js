import { shallow } from 'enzyme';
import messages from "../messages";
describe("navigation component", () => {
    const container = messages;
    it("should render a div", () => {
        expect(container.free).toBe('رایگان');
    });
    it("snapshot" , ()=>{
        expect(container).toMatchSnapshot();
    })
});