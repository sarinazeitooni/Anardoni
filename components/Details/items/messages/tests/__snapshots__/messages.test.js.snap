// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`navigation component snapshot 1`] = `
Object {
  "age": "سن",
  "backBtn": "برگشت",
  "close": "بستن",
  "description": "توضیحات",
  "feedback": "بازخورد",
  "free": "رایگان",
  "getbtn": "دریافت",
  "information": "اطلاعات",
  "linkTitle": "لینک های مفید",
  "more": "بیشتر",
  "swipperTitle": "پیش نمایش",
  "version": " نسخه ",
  "versionHistory": "نمایش تاریخچه ورژن ها",
  "whatsNew": "جدید چه خبره؟",
}
`;
